internal/server/iigointernal/orchestration.go

```diff
@@ -55,14 +56,28 @@ var presidentPointer roles.President = nil
 var iigoClients map[shared.ClientID]baseclient.Client
 
 // RunIIGO runs all iigo function in sequence
-func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.Client) (IIGOSuccessful bool, StatusDescription string) {
+func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.Client, gameConf *config.Config) (IIGOSuccessful bool, StatusDescription string) {
+
+	actionCost = gameConf.IIGOConfig
 
 	iigoClients = *clientMap
 
+	// Increments the budget by a constant 100
+	// TODO:- the constant should be retrieved from the rules
+	g.IIGORolesBudget["president"] += 100
+	g.IIGORolesBudget["judge"] += 100
+	g.IIGORolesBudget["speaker"] += 100
```

Comment:
should be enums

Reply:
Done.
Same for the rest of the enum comments